<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>761</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Advanced Chess</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img/img.qrc">
    <normaloff>:/black/Black/b-queen.svg</normaloff>:/black/Black/b-queen.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>9</y>
      <width>511</width>
      <height>581</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
QGroupBox {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border: 2px solid gray;
 border-radius: 5px;
 margin-top: 1ex; 
}
</string>
    </property>
    <property name="title">
     <string>board</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>821</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
    color: rgb(0, 0, 0);
    background-color: rgb(213, 174, 1);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string>loss pieces</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="b_lost_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;br&gt;&lt;font color='#C90'  &gt;&lt;b&gt; Hover the mouse here to update the list of balck lost peice !!&lt;/b&gt;&lt;/font&gt;&lt;br&gt;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;www.&quot; &gt;&lt;span style=&quot; font-weight:600; text-decoration: underline; color:#ff007f;&quot;&gt;Black pieces lost: &lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="w_lost_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;br&gt;&lt;font color='#C90'  &gt;&lt;b&gt; Hover the mouse here to update the list of White lost peice !!&lt;/b&gt;&lt;/font&gt;&lt;br&gt;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;www.&quot; &gt;&lt;span style=&quot; font-weight:600; text-decoration: underline; color:#ff007f;&quot;&gt;White pieces lost:&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_b">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
    color:white;
    background-color: qlineargradient(x1:0, y1:0, x2:1,y2:1, stop: 1 rgba(41,122, 228, 100), stop: 0 rgba(234, 107, 101, 100));
    border-color: rgba(255,255,255,200);
    border-width: 1px;
    border-style: solid;
    padding: 1px 0px 1px 3px; 
}
/*
  QComboBox {
                    border: 1px solid black;
                    border-radius: 3px;
                 }
     
                 QComboBox::drop-down {
                    width: 32px;
                    border-top-right-radius: 3px;
                    border-bottom-right-radius: 3px;
                 }
     
                 QComboBox::down-arrow {
                    image: url(:/utility/start.png);
                    width: 32;
                    height: 32;
                 }
*/</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_w">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox
{
    color:white;
    background-color: qlineargradient(x1:0, y1:0, x2:1,y2:1, stop: 1 rgba(41,122, 228, 100), stop: 0 rgba(234, 107, 101, 100));
    border-color: rgba(255,255,255,200);
    border-width: 1px;
    border-style: solid;
    padding: 1px 0px 1px 3px; 
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>171</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
    color: rgb(0, 0, 0);
<<<<<<< Updated upstream
    background-color: rgb(6, 179, 177);
=======
<<<<<<< HEAD
<<<<<<< HEAD
    background-color: rgb(255,194,11);
=======
    background-color: rgb(6, 179, 177);
>>>>>>> 71dd5cb6e83be13e96c41903760ea1984eea474a
=======
    background-color: rgb(6, 179, 177);
>>>>>>> 71dd5cb6e83be13e96c41903760ea1984eea474a
>>>>>>> Stashed changes
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string>scores :</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="p1_name">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>player 1: (black)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="p1_positive_1">
       <property name="text">
        <string>positive score:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="p1_positive">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: rgb(0, 0, 0);
<<<<<<< Updated upstream
    background-color: rgb(6, 179, 177);
=======
<<<<<<< HEAD
<<<<<<< HEAD
    background-color: rgb(255,194,11);
=======
    background-color: rgb(6, 179, 177);
>>>>>>> 71dd5cb6e83be13e96c41903760ea1984eea474a
=======
    background-color: rgb(6, 179, 177);
>>>>>>> 71dd5cb6e83be13e96c41903760ea1984eea474a
>>>>>>> Stashed changes
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>----------</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="p1_negative_2">
       <property name="text">
        <string>negative score:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="p1_negative">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: rgb(0, 0, 0);
<<<<<<< Updated upstream
    background-color: rgb(6, 179, 177);
=======
<<<<<<< HEAD
<<<<<<< HEAD
    background-color: rgb(255,194,11);
=======
    background-color: rgb(6, 179, 177);
>>>>>>> 71dd5cb6e83be13e96c41903760ea1984eea474a
=======
    background-color: rgb(6, 179, 177);
>>>>>>> 71dd5cb6e83be13e96c41903760ea1984eea474a
>>>>>>> Stashed changes
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>260</y>
      <width>171</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
    color: rgb(0, 0, 0);
    background-color: rgb(105, 179, 107);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string>scores :</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="p2_name">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>player 2: (white)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="p2_positive_1">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>positive score:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="p2_positive">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: rgb(0, 0, 0);
    background-color: rgb(105, 179, 107);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>----------</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="p2_negative_1">
       <property name="text">
        <string>negative score:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="p2_negative">
       <property name="styleSheet">
        <string notr="true">QLabel
{
    color: rgb(0, 0, 0);
    background-color: rgb(105, 179, 107);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}</string>
       </property>
       <property name="text">
        <string>---------</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
<<<<<<< Updated upstream
    <property name="geometry">
     <rect>
=======
<<<<<<< HEAD
<<<<<<< HEAD
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>510</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Timer</string>
    </property>
    <widget class="QTextEdit" name="secondsText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="minutesText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startStopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Rest</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="hundredthsText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
=======
    <property name="geometry">
     <rect>
=======
    <property name="geometry">
     <rect>
>>>>>>> 71dd5cb6e83be13e96c41903760ea1984eea474a
>>>>>>> Stashed changes
      <x>680</x>
      <y>510</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Timer</string>
    </property>
    <widget class="QTextEdit" name="secondsText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="minutesText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startStopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Rest</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="hundredthsText">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
<<<<<<< Updated upstream
=======
<<<<<<< HEAD
>>>>>>> 71dd5cb6e83be13e96c41903760ea1984eea474a
=======
>>>>>>> 71dd5cb6e83be13e96c41903760ea1984eea474a
>>>>>>> Stashed changes
      <x>11</x>
      <y>11</y>
      <width>121</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="game_label">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic; text-decoration: underline; color:#00aaff;&quot;&gt;Game Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="game_label2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>        -----------</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(160, 226, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>Start New Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redo_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(160, 226, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>Redo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="undo_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(160, 226, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>Undo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dispen_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(160, 226, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>Dispensing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="double_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(160, 226, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>doual move</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit_btn">
       <property name="toolTip">
        <string>
&lt;br&gt;&lt;font color='#C90'  &gt;&lt;b&gt;press this key to exit&lt;/b&gt;&lt;/font&gt;&lt;br&gt;



</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(160, 226, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
<<<<<<< Updated upstream
=======
<<<<<<< HEAD
<<<<<<< HEAD
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>111</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="view_last_btn">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(226,126, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>view last game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_btn">
       <property name="styleSheet">
        <string notr="true">QPushButton
{
height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(226,126, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>load a game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
height:25px;
    color: rgb(0, 0, 0);
    background-color: rgb(226,126, 255);
    border: 1px solid rgb(60, 60, 60);
    border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
=======
>>>>>>> 71dd5cb6e83be13e96c41903760ea1984eea474a
=======
>>>>>>> 71dd5cb6e83be13e96c41903760ea1984eea474a
>>>>>>> Stashed changes
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionsetting"/>
    <addaction name="actionAbout_us"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsetting">
   <property name="text">
    <string>setting</string>
   </property>
  </action>
  <action name="actionAbout_us">
   <property name="text">
    <string>About us</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img/img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>exit_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
